webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/campaigns/MainCampaignService.scala

```diff
@@ -175,14 +192,14 @@ class MainCampaignService(repo: CampaignEventRepository, campaignRepo: CampaignR
         save <- repo.saveCampaignEvent(newCampaign)
         post <-
           newCampaign.state match {
-            case Finished =>
+            case Finished|Skipped(_)  =>
               for {
                 campaign <- campaignRepo.get(event.campaignId)
                 up <-  scheduleCampaignEvent(campaign)
               } yield {
                 up
               }
-            case Scheduled|Running|Skipped =>
+            case Scheduled|Running =>
```

Comment:
we are going to need to draw the transition table for that, I'm sure we are going to misse edge else.